recipes/opendrift/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "opendrift" %}
+{% set version = "1.9.1rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opendrift-{{ version }}.tar.gz
+  sha256: 6f106518d6547759c2cabf43710de8319d3ec202b31159ea6e3b76dbd9f47a7e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python >=3.6,<3.12
+    - pip
+    - poetry
```

Comment:
You probably don't need a build section for a `noarch` package< `host` should do it.
```suggestion
```

Reply:
Thanks, committed.